src/styles/List.js

```diff
@@ -19,6 +19,24 @@ const Container = styled.ul`
     color: ${colors.lightGrey};
     background-color: ${colors.lightestGrey};
   }
+
+  .participants {
+    height: 15px;
+    border-radius: 10px;
+    vertical-align: middle;
+    margin-right: 0;
```

Comment:
instead of 3 separate margin assignments, the short syntax would be good for performance